inventory.py

```diff
@@ -0,0 +1,12 @@
+class Inventory:
+
+    in_inventory = ['Raincoat (worn)', 'Padlock key', 'Parcel']
+
+    @staticmethod
+    def take(pack, item):
```

Comment:
One thing to consider when designing classes is to think about what the code looks like when the class is used. In this case, you'll likely have code like:
```python
inventory = Inventory()
inventory.take("New Item")
```
That reads like the inventory is taking something...which makes less sense than if the method were called add.
```python
inventory = Inventory()
inventory.add("New Item")
```

Reply:
I will fix this.  My working inventory feature is in the while loop, I was debating to use it as a class and kept this file in case it would work better as changes were updated.  Would using a class for inventory be better than using inventory as a regular list in the long run?